.github/workflows/unitctl.yml

```diff
@@ -32,15 +38,21 @@ jobs:
           - build: linux-x86_64
             os: ubuntu-latest
             target: x86_64-unknown-linux-gnu
-          - build: macos-x86_64
+          - build: macos-aarch64
             os: macos-latest
-            target: x86_64-apple-darwin
+            target: aarch64-apple-darwin
```

Comment:
Doesn't this change belong in the previous commit?

Reply:
Personally, I think it should all be squashed into one commit. 
I also am not sure how to add existing in-tree changes into a commit that is not HEAD

Reply:
Okay! I can make sure to have presentation in order before making future merge requests.